backend/app/api/v1/admin.py

```
@@ -32,4 +35,48 @@ async def update_user(
+            version=version,
+            description=description,
+            content=content,
+            is_active=is_active,
+        )
+
+        return await admin_service.create_report_template(
+            db=db,
+            template_data=template_data,
+            user_id=admin.id,
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)